packages/use-intl/src/useTranslations.tsx

```diff
@@ -154,9 +154,11 @@ export default function useTranslations(namespace?: string) {
       }
       const messages = messagesOrError;
 
+      const cacheKey = `${namespace}.${key}`
```

Comment:
The `namespace` is optional. Can you use `[namespace, key].filter(part => part != null).join('.')` here?